src/components/DropdownList/DropdownList.tsx

```diff
@@ -0,0 +1,33 @@
+import { Person } from '../../types/Person';
+import './DropdownList.scss';
+
+type Props = {
+  peoples: Person[] | null;
+  onSelect: (person: Person) => void;
+};
+
+export const DropdownList: React.FC<Props> = ({
+  peoples,
+  onSelect,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        { peoples
+          ? peoples.map((man, index) => (
+            <div
+              key={`${man.name}-${man.born}`}
+              className="dropdown-item"
+              onClick={() => onSelect(man)}
+              onKeyDown={() => onSelect(man)}
```

Comment:
The onKeyDown event handler should not trigger onSelect function as it does not provide a specific key condition, it will fire on any key press. It's recommended to handle specific key events like 'Enter' or 'Space'.